### PR TITLE
PLAT-24395: Add admin console support for password structure multi regex

### PR DESCRIPTION
**Pull request by drorsou**

### CODE REVIEW

File: admin_console/forms/PartnerConfiguration.php [hunk 1]
```diff
@@ -359,6 +359,17 @@ public function init()
...
 		));
 		
+		$this->addElement('button', 'password_structure_validations_edit', array(
+			'label'		=> 'Edit Rules',
+			'decorators'	=> array('ViewHelper'),
+		));
+		$this->getElement('password_structure_validations_edit')->setAttrib('onClick', 'openPasswordValidationRules()');
+		
+		$this->addElement('hidden', 'password_structure_validations', array(
+			'label'		=> 'password_structure_validations',
```
Comment (inbal-ben-david) [inline admin_console/forms/PartnerConfiguration.php:369]: why the label is not written as simple text?

File: admin_console/web/js/assignPasswordRegex.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+function passwordValidationRulesSaveButtonPressed()
+{
+	let regexObject = jQuery.parseJSON("{}");
+	jQuery("fieldset[class=regex-rule]").each(function() {
+		let regex = jQuery(this).find("input[id=regex]").val();
+		let description = jQuery(this).find("input[id=description]").val();
+		regexObject[regex] = description;
```
Comment (inbal-ben-david) [inline admin_console/web/js/assignPasswordRegex.js:7]: it happens only if regex was entered?

File: admin_console/forms/PartnerConfiguration.php (hunk 1 above)
Comment (drorsou) [reply admin_console/forms/PartnerConfiguration.php:369]: Label unused, can be removed

File: admin_console/web/js/assignPasswordRegex.js (hunk 2 above)
Comment (drorsou) [reply admin_console/web/js/assignPasswordRegex.js]: Occurs after clicking 'Save' in the 'Edit Rules' window

**Review by inbal-ben-david — APPROVED** [review]

**Pull request merged**